### ISSUE TITLE
Timeout setting broken in v0.9?

### ISSUE THREAD

**Issue by chipjacks** [taf2/curb#278]
I'm having trouble getting the `Curl::Easy` timeout setting to work in v0.9. It works as expected in v0.8.8 though.

curb_timeout_test.rb:
```ruby
gem 'curb', "= #{ARGV.first}"
require 'curb'
c = Curl::Easy.new("https://httpbin.org/drip?duration=3&numbytes=5&code=200")
c.timeout = 1
puts c.perform
```

Running from the command line:
```
$ ruby curb_timeout_test.rb 0.8.8
/Users/chipjackson/.rvm/gems/ruby-2.0.0-p598/gems/curb-0.8.8/lib/curl/easy.rb:72:in `perform': Timeout was reached (Curl::Err::TimeoutError)
    from curb_timeout_test.rb:5:in `<main>'
$ ruby curb_timeout_test.rb 0.9.0
true
```

**Comment by taf2** [comment]
this looks like it's regression with the addition of the timeout_ms feature...

**Comment by taf2** [comment]
A fix should be available shortly as release 0.9.1
